src/app/_components/keyvisual/index.jsx

```diff
@@ -0,0 +1,34 @@
+import styles from './index.module.scss'
+
+export const Keyvisual = () => {
+  return (
+    <section>
+      <div className={styles['keyvisual-container']}>
+        <img
+          src='/keyvisual/keyvisual-sp.png'
+          alt='キービジュアル'
```

Comment:
[imo]
背景画像のようにコンテンツの文脈上意味を持たない装飾目的の画像は、

```suggestion
          alt=''
```

のように空にすると良いです！

スクリーンリーダーなどの支援技術がalt属性の値を読み上げるのですが、`alt=''`にしてあげれば、文章構造的に意味のない画像と解釈され、何も読み上げなくなります。
背景画像が文章構造上意味を持つことはほとんどないので、背景画像にimgタグを使用する場合は、基本的に`alt=''`で問題ないと思います！

Reply:
ご指摘ありがとうございます！
修正いたしました！

[0f72dc8](https://github.com/gizumo-education/front-coding/pull/229/commits/0f72dc8bf290ac1e14c8aaffd090bb22b148013c)